src/main/java/com/artipie/vertx/VertxSliceServer.java

```diff
@@ -76,7 +77,7 @@
      * @param served The slice to be served.
      */
     public VertxSliceServer(final Vertx vertx, final Slice served) {
-        this(vertx, served, 0);
+        this(vertx, served, new HttpServerOptions().setPort(0));
```

Comment:
Do we really need to set it to zero manually? It seems it'll be zero by default

Reply:
The constructor of options sets 80, so we need to set zero manually.
```
public HttpServerOptions() {
    super();
    init();
    setPort(DEFAULT_PORT); // We override the default for port
}
```